src/Robot.cpp

```diff
@@ -51,6 +52,7 @@ void Robot::DisabledPeriodic() {
   // Scheduler must start running before doing any operations
   // on subsystems or commands.
   Shooter::getInstance()->GetSetShooterCommand()->SetSpeed(0.0);
+  Chassis::getInstance()->GetTankDriveCommand()->TankDrive(true);
```

Comment:
The commit title says "Set Autoshifting to true when robot enabled". This statement is in `Robot::DisabledPeriodic()`. 

I don't think we need this in any case. `TankDrive` is the default command for the chassis.